src/basic-pool-test.sol

```diff
@@ -75,7 +75,7 @@ contract BasisPoolTest is Test {
         vm.startPrank(address(rootContract));
         feed.file(
             "riskGroup",
-            0, // riskGroup:       0
+            1001, // riskGroup:       0
```

Comment:
I don't really understand this change. Also since the test that uses this seems to use 0: https://github.com/centrifuge/tinlake-actions/blob/b010c568a8f9990387b7772886de32083b33b5d4/src/actions.t.sol#L118